Automattic/sensei
# Output logs for failed migration jobs

Resolves #7320 

## Proposed Changes
* Output errors for failed migration jobs
* Make the sync checkbox available for changes: make it possible to restart the migration.

## Testing Instructions

1. Enable sync on a large database and wait for a failed job. (Currently, the branch doesn't include changes from https://github.com/Automattic/sensei/pull/7333 is not merged yet.)
2. Make sure you see logs for the failed job.
3. Try to restart the migration.

## Pre-Merge Checklist


![CleanShot 2023-11-27 at 23 13 35@2x](https://github.com/Automattic/sensei/assets/329356/907f9790-ce03-4f6f-91f7-b089e2f8ad55)
